gateway/server.go

```diff
@@ -348,6 +349,8 @@ func (gw *Gateway) setupGlobals() {
 				Store: &store,
 			}
 			purger.Connect()
+			// register with gob so we can transmit [][]byte
+			gob.Register([][]byte{})
```

Comment:
Why do we need to register a new type to gob?

Reply:
in order that it can  encode and decode a [][]byte datatype,  we need to register this specific type before any encode/decode procesing